### PR TITLE
feat: handle if all tnc is empty 

### PR DESCRIPTION
**Pull request by hanggarHungry24**
ticket basecamp : https://3.basecamp.com/5190892/buckets/31126446/todos/5907450015

### CODE REVIEW

File: stores/booking.ts [hunk 1]
```diff
@@ -86,7 +88,35 @@ const useBookingStore = defineStore("booking", {
...
     };
   },
-  getters: {},
+  getters: {
+    hasTermAndCondition() {
+      const restaurantStore = useRestaurantStore();
+      const bookingPackageStore = useBookingPackageStore();
+      const { selectedPackages } = storeToRefs(bookingPackageStore);
+      const { tncImageUrl } = restaurantStore;
+      if (isProxy(selectedPackages.value[0])) {
```
Comment (afif-hh) [inline stores/booking.ts:97]: isProxy buat apa mas ?

Comment (hanggarHungry24) [reply stores/booking.ts]: udah di ganti di commit terbaru mas , ga butuh getters lagi

File: partial/booking/BookingTnc.vue [hunk 2]
```diff
@@ -64,16 +64,17 @@ import { EVENT_ON_TNC_CLICKED } from "~/composables/hybrid/eventManager";
...
+import { useReport } from "~~/composables/useReport";
 import { storeToRefs } from "pinia";
-import { ref, watch } from "vue";
+import { ref, watch, computed } from "vue";
 const { showGallery } = popUpGallery();
 const { isDesktop } = storeToRefs(useClientTypeStore());
 const { tncImageUrl } = storeToRefs(useRestaurantStore());
 const { selectedPackages } = storeToRefs(useBookingPackageStore());
-const { hasTermAndCondition } = storeToRefs(useBookingStore());
+// const { hasTermAndCondition } = storeToRefs(useBookingStore());
```
Comment (afif-hh) [inline partial/booking/BookingTnc.vue:77]: dihapus mas

File: partial/booking/BookingTnc.vue [hunk 3]
```diff
@@ -74,7 +74,6 @@ const { showGallery } = popUpGallery();
 const { isDesktop } = storeToRefs(useClientTypeStore());
 const { tncImageUrl } = storeToRefs(useRestaurantStore());
 const { selectedPackages } = storeToRefs(useBookingPackageStore());
-// const { hasTermAndCondition } = storeToRefs(useBookingStore());
```
Comment (hanggarHungry24) [inline partial/booking/BookingTnc.vue]: udah dihapus mas @afif-hh

**Review by afif-hh — APPROVED** [review]

**Pull request merged**